babel.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = function (api) {
       '@babel/plugin-transform-runtime',
       '@babel/plugin-proposal-class-properties',
       '@babel/plugin-proposal-object-rest-spread',
+      '@babel/plugin-proposal-optional-chaining'
```

Comment:
```suggestion
      '@babel/plugin-proposal-optional-chaining',
```
This should fix the lint failure